packages/middleware-retry/package.json

```diff
@@ -15,6 +15,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@aws-sdk/service-error-classification": "1.0.0-gamma.1",
+    "@aws-sdk/smithy-client": "1.0.0-gamma.1",
```

Comment:
Looks like the only imported `SdkError` and `RetryableTrait` are interfaces, should we move it to `devDependencies`?

Reply:
Moved in [commit](https://github.com/aws/aws-sdk-js-v3/pull/1238/commits/74bb5d79cdc7b3adb48f06a55837cc6b87e2c109)